index.html

```diff
@@ -12107,17 +12107,17 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</ol>
-				<p class="note">Note that <a>Assistive Technologies</a> with Braille support can convert <code>aria-roledescription</code> content to Braille. In addition, assistive technologies will be able to customize such Braille output according to user preferences. Using only <code>aria-roledescription</code> is <strong>almost always</strong> the better user experience and authors are <strong>strongly discouraged</strong> from using <code>aria-brailleroledescription</code> to replicate <code>aria-roledescription</code>. Instead, <code>aria-brailleroledescription</code> is meant be used only when <code>aria-roledescription</code> cannot provide an adequate Braille representation, i.e., when a specialized Braille description is very different from a text description converted to Braille. It is very important to note that when using <code>aria-brailleroledescription</code> authors are solely responsible to align the attribute value with the document language and clearly communicate the use of this attribute to the user. This is even more important when the value consists of Unicode Braille Patterns because <a>Assistive technologies</a> will pass such content directly to the user without applying user specific Braille translations; in general, authors are <strong>strongly discouraged</strong> from using Unicode Braille Patterns in <code>aria-brailleroledescription</code>.
+				<p class="note">Note that <a>Assistive Technologies</a> with braille support can convert <code>aria-roledescription</code> content to Braille. In addition, assistive technologies will be able to customize such braille output according to user preferences. Using only <code>aria-roledescription</code> is <strong>almost always</strong> the better user experience and authors are <strong>strongly discouraged</strong> from using <code>aria-brailleroledescription</code> to replicate <code>aria-roledescription</code>. Instead, <code>aria-brailleroledescription</code> is meant be used only when <code>aria-roledescription</code> cannot provide an adequate braille representation, i.e., when a specialized braille description is very different from a text description converted to Braille. It is very important to note that when using <code>aria-brailleroledescription</code> authors are solely responsible to align the attribute value with the document language and clearly communicate the use of this attribute to the user. This is even more important when the value consists of Unicode Braille Patterns because <a>Assistive Technologies</a> will pass such content directly to the user without applying user specific braille translations; in general, authors are <strong>strongly discouraged</strong> from using Unicode Braille Patterns in <code>aria-brailleroledescription</code>.
```

Comment:
The following adds a missing comma, corrects some other grammar, and makes minor wording changes that hopefully improve clarity.

```suggestion
				<p class="note"><a>Assistive technologies</a> that support braille output can convert <code>aria-roledescription</code> values to Braille. In addition, assistive technologies may also apply user preferences, such as for a specific grade of braille,  to the conversion. Using only <code>aria-roledescription</code> will <strong>almost always</strong> provide the best user experience, and authors are <strong>strongly discouraged</strong> from using <code>aria-brailleroledescription</code> to replicate <code>aria-roledescription</code>. Instead, <code>aria-brailleroledescription</code> is meant to be used only when <code>aria-roledescription</code> cannot provide an adequate braille representation, i.e., when an ideal braille description would be very different from a text description converted to Braille. It is very important to note that when using <code>aria-brailleroledescription</code>, authors are solely responsible for aligning the attribute value with the document language and clearly communicating use of this attribute to the user. This is even more important when the value consists of Unicode Braille Patterns because <a>Assistive Technologies</a> will pass such content directly to the user without applying user specific braille translation preferences. In general, authors are <strong>strongly discouraged</strong> from using Unicode Braille Patterns in <code>aria-brailleroledescription</code>.
```

Reply:
I had missed this earlier. Since @mcking65 and @cookiecrook have indicated approval of this version, I will leave this as proposed here (i.e., I will **not** replace it with @joanmarie 's version as I wrote in an earlier comment of mine). I hope that's ok with @joanmarie as well.

Reply:
But I worked so hard on it. :stuck_out_tongue: 
If everyone else is happy with your revisions, then I'm happy too.